src/oc_erchef/schema/deploy/users_email_functional_index.sql

```diff
@@ -0,0 +1,10 @@
+-- Deploy enterprise_chef:users_email_functional_index to pg
+-- requires: users
+
+BEGIN;
+
+  -- TODO 2017/05/30 sr: check with ops if this should include CONCURRENTLY
+  -- https://www.postgresql.org/docs/9.2/static/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY
+  CREATE INDEX users_lower_email_idx ON users (lower(email));
```

Comment:
Unfortunately, I think CREATE INDEX IF NOT EXISTS isn't supported in 9.2.  This stack-overflow answer presents alternatives for different versions: